vmm/src/config.rs

```diff
@@ -3690,6 +3690,219 @@ mod tests {
         Ok(())
     }
 
+    #[test]
```

Comment:
Also let's squash this commit with the previous one that added the related parsing and validation function.